external/wakaama/examples/client/lwm2mclient.c

```diff
@@ -1185,7 +1187,7 @@ pthread_addr_t client_main(void)
 
 		if (NULL == pskBuffer) {
 			fprintf(stderr, "Failed to create PSK binary buffer\r\n");
-			return -1;
+			return NULL;
```

Comment:
Dear dr-venkman
Thanks for reviewing. We fully agreed your opinion and now we fix it.
"-1" returns in error case and "0" returns in normal case.